### PR TITLE
makes sure pgbouncer is restarted after reboot

### PR DESCRIPTION
**Pull request by javierwilson**

### CODE REVIEW

File: ansible/roles/postgresql/tasks/main.yml [hunk 1]
```diff
@@ -180,6 +180,16 @@
   tags:
     - pgbouncer
 
+- shell: /bin/true
+  register: no_tags
```
Comment (kaapstorm) [inline ansible/roles/postgresql/tasks/main.yml]: This is a nice approach! Are there other things that need to happen once, that we can use this with too?

Comment (snopoke) [inline ansible/roles/postgresql/tasks/main.yml]: can you elaborate on whet this is for?

Comment (javierwilson) [reply ansible/roles/postgresql/tasks/main.yml]: this will only be run when using no tags, this allows to run tasks only when a specific tag is used.

Comment (snopoke) [reply ansible/roles/postgresql/tasks/main.yml]: interesting, I think it's worth a comment (maybe even link to the SO answer: https://serverfault.com/a/748864)

Comment (emord) [reply ansible/roles/postgresql/tasks/main.yml]: I think this is causing pgbouncer to restart unconditionally (but not during `--check`). I ran deploy_db with `--tags=pgbouncer` and in check mode it said this restart was skipped, but when I ran without `--check` pgbouncer was restarted. I haven't figured it out why that is though :/

Comment (snopoke) [reply ansible/roles/postgresql/tasks/main.yml]: https://github.com/dimagi/commcare-cloud/pull/3151

**Pull request merged**